webapp/src/app/(dashboard)/home/page.tsx

```diff
@@ -55,7 +57,37 @@ export default async function HomePage({
                 <CardHeader>
                     <CardTitle>Get Started</CardTitle>
                     <CardDescription>
-                        Create your first organization to begin
+                        Create your first organization to begin. <br />
```

Comment:
Do we want to force org creation to start ? Alternative suggestion : 
`Publish data to your personal organization to begin.`

Reply:
I just removed the line. It was not really useful.